CRM/Xcm/Tools.php

```diff
@@ -39,7 +39,7 @@ public static function getAddressFields() {
    * return all contact detail fields
    */
   public static function getDetailFields() {
-    return ['email', 'url', 'phone'];
+    return ['email', 'url', 'phone', 'website'];
```

Comment:
I don't get why that is necessary in this context. @bjendres does this have implications regarding matching and/or filling fields?